### ISSUE TITLE
requirements.txt problem...

### ISSUE THREAD

**Issue by zeek-han** [jefflai108/pytorch-kaldi-neural-speaker-embeddings#4]
Hi, first of all, I really appreciate for your work based on KALDI platform for "Exploring the Encoding Layer and Loss Function in End-to-End Speaker and Language Recognition System".

but, I have some trouble when I first start your code...
When I execute "pip install -r requirements.txt",  an error occurs.
The error message is 

> Invalid requirement: '_libgcc_mutex=0.1=main'
> = is not a valid operator. Did you mean == ?

and, honestly, I've never seen the unary operator '=' in requirements.txt
I wonder why the unary operator exists there.
In almost every line of "requirements.txt", there is '=' .

Then...how do I execute the code? 
please help me...
Thank you in advance
